src/main/scala/dpla/ingestion3/mappers/providers/NaraMapping.scala

```diff
@@ -247,22 +247,34 @@ class NaraMapping extends XmlMapping with XmlExtractor {
     uri = Some(URI("http://dp.la/api/contributor/nara"))
   )
 
-  private def extractCollection(data: NodeSeq): Seq[String] = {
+  private def extractCollection(data: NodeSeq): Seq[DcmiTypeCollection] = {
     val parentRecordGroupIds = for {
       prg <- data \\ "parentRecordGroup" \\ "title"
-    } yield prg.text
+      prgUri <- extractString(data \\ "parentRecordGroup" \\ "naId")
```

Comment:
Not the end of the world, but it's probably more idiomatic to use `=` in a `for` rather than chaining `map`.